httpbakery/form/client.go

```diff
@@ -0,0 +1,201 @@
+package form
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/juju/schema"
+	"golang.org/x/crypto/ssh/terminal"
+	"gopkg.in/errgo.v1"
+	"gopkg.in/juju/environschema.v1"
+)
+
+// PromptingFiller implements a Filler by prompting for each field in
```

Comment:
It would probably be nice to start with a general description what we aim to do with this package.